### ISSUE TITLE
Policy change proposal - json IXP data

### ISSUE THREAD

**Issue by durkovic** [peeringdb/peeringdb#308]
I'd like to discuss the policy regarding creation of new entries in peeringdb
from automatic IXP exports in json format.

Currently, peeringdb does not create missing entries and basically trashes
first-hand verified data from IXPs. As a result, the number of participants at
IXPs significantly differ from reality.

While I understand that peeringdb historically preferred network-provided data,
with automation and unified json format in place at several IXPs, it's a pity
to just trash this data and don't use it for the community.

Therefore I'd like to propose a policy change regarding the creation of new
entries - for instance, peeringdb could create new entries from IXP exports and
flag them as originating from IXP, in order to allow networks to distinguish
them from network-provided data. That way, the reliable data would no longer get
lost, but instead could be used by all tools developed for peeringdb.

**Comment by grizz** [comment]
@peeringdb/pc no votes on this, +1 from me. This would require a board decision AFAIK, since it's against policy to create entries without a network explicitly adding it.

@bijals is working on a similar proposal.

Moving it back to review pending discussion and board decision.

**Comment by mcmanuss8** [comment]
Which cases are we trying to solve for here?

1) The network doesn't exist in peeringdb at all
2) The network exists in peeringdb but doesn't have a record at this IX
3) The network exists in peeringdb and has a record at the IX but it's wrong (different IP[s] or speed?)

For case 1, that's a bit tricky - there's a lot of fields we wouldn't know how to fill in (there's no org, contact info, etc). I don't think we can handle that at all other than to email the whois record for the ASN and suggest they sign up for peeringdb?

For cases 2 and 3, that's certainly doable, we just need the network to sign off on allowing automation to update their records for them.

**Comment by arnoldnipper** [comment]
@grizz , adding a network w/o the owner explicitly did that happens frequently. Why? A lot of people do not really know what they do. The network is added, but admin status is not granted as there is a mismatch. So AC is mailing the owner and immediately closing the ticket to keep the queues clean. Only if the owner mails back there is further action.

**Comment by grizz** [comment]
I think @durkovic is explicitly asking for the IX-F entries to show up whether the network wants them to at all, which would need a board decision

for 1, I wasn't aware that it happens now, it should be fairly easy to just create a simple entry (we can get the information from RDAP), and import the IX-F feed for the network, but that should be a new ticket, @peeringdb/pc should we create an issue for that?

Thanks @mcmanuss8 and @arnoldnipper for the replies

**Comment by durkovic** [comment]
@mcmanuss8 my proposal covers all 3 cases, since it's not uncommon to see significantly less networks appearing in peeringdb than really connected to an IX. And lot of entries are not uptodate, since many networks simply don't spend time to maintain their records manually.

Entries created from IX-F feed could be perhaps marked by some special flag to distinguish them from network-provided data.

**Comment by durkovic** [comment]
OK, let's try to split the problem, since 95 % of missing entries in our IXP are due to 1 (network doesn't exist in peeringdb at all). Solving 1 would therefore significantly improve the situation.

So would it be possible to follow @grizz advice i.e. create simple network entry from RDAP in such situation and import the IX-F feed for this network?

**Comment by grizz** [comment]
@durkovic Yes, new ticket for that, I like the idea!

As for this, it's still waiting on a board decision.

**Comment by arnoldnipper** [comment]
@durkovic, I guess we can get has been implemented by the new IX-F JSON importer. A network gets notified if the IX-F JSON indicates that it is connected, however, a netixlan record is missing. Do you agree? If so, please close the issue. If not we will escalate.

**Comment by arnoldnipper** [comment]
@durkovic, please open a new issue as proposed by @grizz. Closing here
